src/views/HomeIndex.vue

```diff
@@ -35,12 +35,17 @@ const deleteConfirmation = (nodeUuid) => {
 const handleNodeNotFound = () => {
   router.push({ name: 'node.not-found' });
 };
+
+const handleSaveStore = () => {
+  saveStore();
+};
```

Comment:
this could be a computed property that creates the link using the nodes state from the store

```
const downloadNodesLink = computed(() => {
  const jsonNodes = JSON.stringify(nodes.value);
  const blob = new Blob([jsonNodes], { type: 'application/json' });
  return URL.createObjectURL(blob);
});
```

no need to create a `a` element and append to the body as the `Button` element can already do that.